README.md

```diff
@@ -104,7 +104,7 @@ You're a good person and decide to validate your HTML. You copy and paste the co
 The validator throws errors at you! Why? Assuming you haven't made any mistakes in your code, how could you go about accurately validating your HTML?
 
 ```
-Your answer...
+The "clown hats" wrapping the content are ruby symbols, not HTML, which is why the validator throws the error. To accurately validate, I would either remove the clown hat symbols or paste the code into a ruby validator online.
```

Comment:
You say you need valid HTML, If you remove the clown hats would it create valid HTML?  
Please Slack me the ruby validator you use.  My interest is piqued.
Please review our suggestion.